lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -4947,6 +4947,29 @@ describe('workers/repository/process/lookup/index', () => {
       ]);
     });
 
+    it('handles replacements - from datasource', async () => {
+      config.currentValue = '2.0.0';
+      config.packageName = 'org.example:foo';
+      config.datasource = MavenDatasource.id;
+      getMavenReleases.mockResolvedValueOnce({
+        releases: [{ version: '2.0.0' }],
+        replacementName: 'foo:bar',
+        replacementVersion: '2.0.0',
+      });
+
+      const { updates } = await Result.wrap(
+        lookup.lookupUpdates(config),
+      ).unwrapOrThrow();
```

Comment:
```suggestion
      const { updates } = await lookup.lookupUpdates(config);
```
no need to wrap here

Reply:
With your suggestion I receive `Property 'updates' does not exist on type 'Result<UpdateResult, Error>'.ts(2339)`.

The only working code would be the following:
```ts
      const { updates } = (await lookup.lookupUpdates(config)).unwrapOrThrow();
```
As all other tests don't use this variant, I think it shouldn't be changed.